### ISSUE TITLE
country-info consistency: "[formerly] known as..." w/ or w/o year(s)

### ISSUE THREAD

**Issue by horwitz** [anki-geo/ultimate-geography#545]
In `country_info.csv`, Eswatini has the note "Known as Swaziland until 2018.", yet there are other countries with notes of the form "Formerly known as X" (with no year). Consider either of two options:

(1) Change Eswatini's note to "Formerly known as Swaziland" (and change DRC's from "Formerly Zaire" to "Formerly known as Zaire") or

(2) Update other notes to include years—in particular:

- North Macedonia
"Formerly known as Macedonia."
-> "Known as Macedonia from 1991 to 2019."

- Democratic Republic of the Congo
"Formerly Zaire."
-> "Known as Zaire from 1971 to 1997."

- Republic of Artsakh
"Independent state claimed by Azerbaijan, formerly known as Nagorno-Karabakh."
-> "Independent state claimed by Azerbaijan, known as Nagorno-Karabakh from 1991 to 2017."

Two notes for option (2): the years should be checked (I was pretty fast and loose in my initial analysis) and the Eswatini note should probably also be updated (to have not only an end date but also a start date)—i.e., "Known as Swaziland from XXXX to 2018." (for the right value of XXXX).

**Comment by horwitz** [comment]
I read 1 as

"Known as Macedonia (for all time) until 2019." (which is, of course, not true)

whereas I read 3 (for, say, DRC) as

"Known as Zaire from 1971 to 1997 (that's just the last name before the current one—I make no promises about what it was called before then)" (which (I think) _is_ correct)

Arguably, we could try to give all the historical names for each country (as a sovereign/independent nation) but, well, that seems like quite the rabbit hole (that I'm not actually suggesting we fall down!)

(Also, 2 technically doesn't note that this was the _most recent_ previous name—just that it was _some_ previous name.)

Realistically, I don't think people will actually read 1 as if it had a "for all time" clause (even if, technically, that's a reasonable way to read it), but I see much potential confusion in 3 (admittedly, I can appreciate _wanting_ more information (as an Anki user) :) )

3 > 1 > 2 (but no huge dislike for any)

... and 1 could move in my estimation, FWIW, with some rewording (e.g., "Name changed from Macedonia in 2019" no longer has the implication that it was Macedonia from the beginning of time (or at least the beginning of the nation's independence)... though that particular wording may not be optimal...)

**Comment by axelboc** [comment]
All your arguments are valid. 😄 

Personally, I'd be more in favour of doing nothing. The initial idea was to put the year when it was quite recent (though this should definitely be more strictly defined in the guidelines), so users who have pre-existing geography knowledge don't be all like "waat? since when is it called that!? 🤔"

Sure, the wording with the year is not ideal, but it has the merit of being concise. Also, I don't think it's that confusing when the change is relatively recent, since it remains more about geographical trivia than historical trivia.

**Comment by axelboc** [comment]
Yeah, sorry, by "doing nothing", I meant keeping the current two wording options but we should definitely at least use them consistently, maybe with a cut-off time span of, say, 20 years:

```
Known as Macedonia until 2019.
Independent state claimed by Azerbaijan, known as Nagorno-Karabakh until 2017.
```

That being said, I wouldn't mind too much "renamed from" as an alternative wording (over "name changed from"), though I feel like it is still a bit reductive as it separates out the country from its name, as if the two evolved separately. Not sure I'm being clear.

**Comment by horwitz** [comment]
Thanks—that clears some stuff up!

"Renamed from" is certainly better (IMO) than "name changed from"

I don't see the issue with "renamed from" or the like being reductive—but I do mean "I don't see"... _I_ don't see it (there certainly may be something there!) Is this because "Renamed from..." sounds like some external force (i.e., not the renamed country) is doing the renaming?

**Comment by ohare93** [comment]
I agree "Renamed from" gives a connotation of external force changing it's name :thinking: it was the first thing I thought, at least.

**Comment by aplaice** [comment]
Copying from the closed Nursultan/Astana thread:

We _might_ want to treat very temporary changes (several years, where the current and ante-penultimate names are the same) differently from other changes, so that even if we decide to go with just the final change date in most cases, we might want to have a full date range for the "temporary" cases, since it helps avoid confusion (people who remember the name change, but, say, don't remember that there were two name changes might think that the "temporary" name was the old one).

For instance, for Kazakhstan/Astana, we could have something like: "Briefly renamed to Nur-Sultan (2019-2022)." (?).

(I don't have a strong opinion about this, though!)

**Comment by horwitz** [comment]
if going down this path, (1) consider explicit guidelines for what makes a name change temporary (e.g., that the antepenultimate name matches the current one (as @aplaice note), plus an explicit upper bound on time the name was used) and (2) consider phrasing as "Briefly named Nur-Sultan (2019–2022)" (note the en dash in the range description ("2019–2022"))
